snuba/query/__init__.py

```diff
@@ -377,6 +377,11 @@ def transform(self, visitor: ExpressionVisitor[Expression]) -> None:
                 self.__order_by,
             )
         )
+        if self.__limitby is not None:
+            self.__limitby = LimitBy(
+                self.__limitby.limit,
+                [expression.accept(visitor) for expression in self.__limitby.columns],
+            )
```

Comment:
Was there a reason why you couldn't call transform on the whole limit by clause?

i.e. `self.__limitby = self.__limit_by.transform(func) if self.__limitby else None`

Reply:
My understanding is that `Limitby` is a dataclass where only the [columns are expressions](https://github.com/getsentry/snuba/blob/master/snuba/query/__init__.py#L35), and the visitor applies transformation to only expressions.
```
@dataclass(frozen=True)
class LimitBy:
    limit: int
    columns: Sequence[Expression]
```